### ISSUE TITLE
Unable to use Claude 3 with Google Vertex AI

### ISSUE THREAD

**Issue by imdadahad** [langchain-ai/langchainjs#7211]
### Checked other resources

- [X] I added a very descriptive title to this question.
- [X] I searched the LangChain documentation with the integrated search.
- [x] I used the GitHub search to find a similar question and didn't find it.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```typescript

import { ChatVertexAI } from "@langchain/google-vertexai"

    const systemMessage = "Give me a concise answer"
    const promptMessage = "What is the meaning of life?"
    const vertextAIClient = new ChatVertexAI({
        location: "europe-west1",
        model: "claude-3-5-sonnet@20240620",
    })
    const result = await vertextAIClient.invoke([
        promptMessage,
        systemMessage,
    ])
}
```

### Description

I am trying to use Claude 3 Sonnet in Langchain GoogleVertexAI but am receiving the error: "base.getType is not a function"

I've scoured the documentation, searched across the repository and came across this community post where the answer was provided by a bot but was an incorrect suggestion.

### System Info

Platform - Mac
 Node Version - 20.16.0
 NPM - 10.8.1

**Comment by imdadahad** [comment]
^ This is incorrect for a human who may be reading this.

I am trying to use the Claude 3.5 (not the v2) and this is a valid model made available on GCP. Additionally, the suggestion to specify a model ID is incorrect. This is not how it is described on the Langchain documentation and upon trying (for the sake of it), I receive an 'Unable to verify model params' error.

**Comment by bracesproul** [comment]
@imdadahad what version of `@langchain/core` and `@langchain/google-vertexai` are you using? If you update to latest on both does it fix it?

**Comment by imdadahad** [comment]
@bracesproul I am using version `0.2.18` of `@langchain/core` and `0.1.2` of `@langchain/google-vertexai`

I upgraded the core library to `0.3.18` but I am still receiving the same error

**Comment by imdadahad** [comment]
@bracesproul any idea? 

Is this a bug with the library or on GCP's end..

**Comment by bracesproul** [comment]
@imdadahad fix is now out on version `0.1.3`! Please tag me if you run into more issues.

**Comment by sunnyunde-xooa** [comment]
@bracesproul   Getting this error for deepseek-ai/deepseek-r1-0528-maas with Google Vertex AI
I am using following packages
"@langchain/core": "0.3.56",
"@langchain/google-vertexai": "^0.2.0",
Error message for reference 

Unable to verify model params: {"lc":1,"type":"constructor","id":["langchain","chat_models","chat_integration","ChatVertexAI"],"kwargs":{"model_name":"deepseek-ai/deepseek-r1-0528-maas","model":"deepseek-ai/deepseek-r1-0528-maas","temperature":0,"max_output_tokens":768,"streaming":true,"max_retries":4,"auth_options":{"lc":1,"type":"secret","id":["GOOGLE_AUTH_OPTIONS"]},"platform_type":"gcp"}}
